runtime/moonbase/tests/integration_test.rs

```diff
@@ -468,18 +468,18 @@ fn reward_block_authors_with_parachain_bond_reserved() {
 				run_to_block(x, Some(NimbusId::from_slice(&ALICE_NIMBUS).unwrap()));
 			}
 			// no rewards doled out yet
-			assert_eq!(Balances::free_balance(AccountId::from(ALICE)), 1_000 * UNIT,);
-			assert_eq!(Balances::free_balance(AccountId::from(BOB)), 500 * UNIT,);
+			assert_eq!(Balances::free_balance(AccountId::from(ALICE)), 2_000 * UNIT,);
+			assert_eq!(Balances::free_balance(AccountId::from(BOB)), 1_000 * UNIT,);
 			assert_eq!(Balances::free_balance(AccountId::from(CHARLIE)), UNIT,);
 			run_to_block(1200, Some(NimbusId::from_slice(&ALICE_NIMBUS).unwrap()));
 			// rewards minted and distributed
 			assert_eq!(
 				Balances::free_balance(AccountId::from(ALICE)),
-				1082693333281650000000,
+				2082693333281650000000,
 			);
 			assert_eq!(
 				Balances::free_balance(AccountId::from(BOB)),
-				525841666640825000000,
+				1025841666640825000000,
 			);
```

Comment:
Consider adding a check for `usable` balance

Reply:
Thanks for pointing this out, it's quite useful. I feel like `usable_balance` should take a `WithdrawReasons`, though... :thinking:

Reply:
Fixed in ded490e, thanks